### DISCUSSION TITLE
Add a style prop for Background opacity

### DISCUSSION THREAD

**Discussion by ghost** [chakra-ui/chakra-ui#8889] [Ideas]
### Description

I wish for a bgOpacity style prop to be created to allow users set the opacity of the background color for an element.

### Problem Statement/Justification

In bootstrap there is a style name `bg-opacity-25` which can be used to determine the opacity of the background of an element. This will cause the background color to have an opacity of 25. At there moment there is no allocation for this using Chakra.

### Proposed Solution or API

 I believe adding a style prop which accepts an integer to determine the opacity of the background. Could be extremely useful.
Maybe this can use the rgba values used in CSS to get this to work.

**Comment by Urvi-Pandit** [comment]
You may start by extending the Chakra UI theme to include custom 'bgOpacity' styles. These changes can be made in 'chakra.config.js' or 'theme.js'
This is how you may do it
const theme = {
  // ... other theme configurations

  extendTheme: {
    styles: {
      global: (props) => ({
        // ... other global styles
        '.bg-opacity-25': {
          backgroundColor: `rgba(0, 0, 0, 0.25);`, // 25% opacity black background
        },
        // You can define more opacity levels as needed
      }),
    },
  },
};

export default theme;

Here we define the .bg-opacity-25 class to set the background color with 25% opacity. You can add more classes for different opacity levels as needed.
